feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -146,16 +146,26 @@ export class ConfiguratorAttributeBaseComponent {
    * @param expMode - Is expert mode set?
    * @param label - value label
    * @param techName - value technical name
+   * @param value - item Object which contains the valuePrice
+   * @param isSelected - is item in dropdown list selected
```

Comment:
Adjust your comment, there is no `isSelected` parameter

Reply:
done